vinyldns/resources.go

```diff
@@ -50,6 +50,15 @@ type ListFilter struct {
 	MaxItems   int
 }
 
+// ListFilterInt represents the list query parameters that may be passed to
+// VinylDNS API endpoints such as /zones and /zones/${zone_id}/recordsets
+// Same as ListFilter except for resources where StartFrom is an int instead of string
+type ListFilterInt struct {
+	NameFilter string
```

Comment:
I don’t think `ListFilterInt` is used anywhere else other than for recordsetChanges, and it doesn't have any nameFilter as of now. We can remove this for now.